content/en/docs/mechanical-assembly/belt-tensioners/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Belt Tensioners"
+linkTitle: "Belt Tensioners"
+weight: 9
+resources:
+- src: "**.png"
+- src: "**.jpg"
+description: >
+  Assembling the belt tensioners used throughout the machine
+---
+
+Alright, time to get into the build! First, we're going to build the belt tensioners that are used for the X and Y axes. You need to build **three** of these.
+
+| Qty              | Part                          |
+|-----------------:|-------------------------------|
+| 1 each, 3 total  | M3 Hex Nut                    |
+| 1 each, 3 total  | M3x16 machine screw           |
+| 1 each, 3 total  | M3 Cap Nut                    |
+| 1 each, 3 total  | M5x25 machine screw           |
+| 1 each, 3 total  | M5 Nyloc Hex Nut              |
+| 1 each, 3 total  | M5x40 machine screw           |
+| 1 each, 3 total  | M5 Hex Nut                    |
+| 1 each, 3 total  | GT2 Pulley Idler              |
+| 1 each, 3 total  | FDM-0037 (Belt Tension Arm)   |
+| 1                | FDM-0001 (Front Left Leg)     |
+| 1                | FDM-0002 (Front Right Leg)    |
+| 1                | FDM-0013 (X Idler Mount)      |
+
+## Assembling the tensioner arms
+
+{{< container-image path="images/1-belt-tension-arm-parts.jpg" alt="Belt Tension Arm Parts" >}}
+
+1. Insert an M3 Hex Nut into its slot on the back of a {{< tooltip "Belt Tension Arm" "FDM-0037" >}}.
```

Comment:
FYI I fixed the numbering to be consistent. Now it counts up and continues passed the ## headers. I'm not 100% sure I love this, especially because my linter cries when I do this, but it was inconsistent before on this page, and now it is how I was trying to do it for the other pages. We can obviously have a discussion on switching the whole site back to individual ordered lists if you think it's worth it.